code/game/machinery/suit_storage_unit.dm

```diff
@@ -53,11 +53,11 @@
 	/// Cooldown for occupant breakout messages via relaymove()
 	var/message_cooldown
 	/// How long it takes to break out of the SSU.
-	var/breakout_time = 300
+	var/breakout_time = 30 SECONDS
 	/// Power contributed by this machine to charge the mod suits cell without any capacitors
-	var/base_charge_rate = 200
+	var/base_charge_rate = 20 KILO JOULES
```

Comment:
```suggestion
	var/base_charge_rate = 200 KILO WATTS
```
It gets multiplied by time, so you use watts. Also cells got scaled by 1,000, so we scale by 1,000 instead of 100.